resources/launch.json

```diff
@@ -0,0 +1,12 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Run Camel application and attach Camel debugger",
```

Comment:
maybe I should mention `native` also here?

Reply:
WDYT @apupier ?

Reply:
agree